lib/plugins/aws/package/compile/events/rabbitmq.js

```diff
@@ -45,9 +45,12 @@ class AwsCompileRabbitMQEvents {
         queue: {
           type: 'string',
         },
+        virtualHost: {
+          type: 'string',
+        },
       },
       additionalProperties: false,
-      required: ['basicAuthArn', 'arn', 'queue'],
+      required: ['basicAuthArn', 'arn', 'queue', 'virtualHost'],
```

Comment:
Maybe the `virtualHost` field should be an optional?

Reply:
updated in latest commit